01_temperature/temperature.rb

```diff
@@ -1 +1,15 @@
-#write your code here
+def  ftoc(temperature)
+   
+  (temperature -32.0)  * 5/9
+ 
+
+end
+
+def  ctof(celsius)
+   
+   
+  ( celsius * 9.0) / 5.0 + 32
+  
+    
+
```

Comment:
Why is there so much empty space here?

```suggestion
  (celsius * 9.0) / 5.0 + 32
```